### PR TITLE
DENG 1022 - Update bhr_collection DAG to write to GCS instead of S3

### PR DESCRIPTION
**Pull request by alekhyamoz**
TODO - 
1. The GCS bucket in this PR points to `moz-fx-data-static-websit-f7e0-analysis-output`. Need to find out the right bucket
2. Create a subsequent PR in `telemetry-airflow` to update the dag config and remove the gcs_sync task

### CODE REVIEW

**Review by akkomar — APPROVED** [review]
LGTM.
> The GCS bucket in this PR points to moz-fx-data-static-websit-f7e0-analysis-output. Need to find out the right bucket

Production bucket is mentioned in the doc attached in Jira issue.

**Comment by alekhyamoz** [comment]
> LGTM.
> 
> > The GCS bucket in this PR points to moz-fx-data-static-websit-f7e0-analysis-output. Need to find out the right bucket
> 
> Production bucket is mentioned in the doc attached in Jira issue.

As per, https://github.com/mozilla/telemetry-airflow/blob/main/dags/bhr_collection.py#L122, moz-fx-data-static-websit-8565-analysis-output seems to be the bucket.

**Comment by kik-kik** [comment]
Hey @alekhyamoz, what's the current status of this change? Is there anything else that needs to be done or could this be merged?

File: mozetl/bhr_collection/bhr_collection.py [hunk 1]
```diff
@@ -1238,6 +1252,8 @@ def start_job(date, sample_size):
             "hang_lower_bound": 128,
             "hang_upper_bound": 65536,
             "sample_size": sample_size,
+            "use_gcs": use_gcs,
+            "use_s3": not use_gcs,
```
[the remark this answers is not in the document]
Comment (kik-kik) [reply mozetl/bhr_collection/bhr_collection.py:1256]: Could we not just re-run the task instance once the bucket is updated to the prod bucket?

**Review by kik-kik — APPROVED** [review]

File: mozetl/bhr_collection/bhr_collection.py
Comment (kik-kik) [inline mozetl/bhr_collection/bhr_collection.py:1]: Maybe we should at a module level comment explaining that the s3 logic is there for backwards compatibility only due to the AWS to GCP migration?

**Pull request merged**